release_notes.md

```diff
@@ -17,3 +17,4 @@
 - Ordered invocations are now the default (#10201)
 - Skip worker description if none of the profile conditions are met (#9932)
 - Fixed incorrect function count in the log message.(#10220)
+- Adding a timeout to `GetFunctionMetadataAsync` to prevent deadlocks (#10219)
```

Comment:
Can we update the note here so it's more end user friendly (i.e., doesn't require understanding of the codebase). Something along the lines of `Adding a timeout when retrieving function metadata from metadata providers`

Reply:
Updated as suggested.